### PR TITLE
[ADD] allow to clean up menus

### PR DESCRIPTION
**Pull request by hbrunn**
why in 7.0 api? Because then it's simple to do the same for <= 7.0

### CODE REVIEW

**Comment by pedrobaeza** [comment]
:+1:

File: database_cleanup/model/purge_menus.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        'menu_id': fields.many2one('ir.ui.menu', 'Menu entry'),
+    }
+
+    def purge(self, cr, uid, ids, context=None):
+        self.pool['ir.ui.menu'].unlink(
+            cr, uid,
+            [this.menu_id.id for this in self.browse(cr, uid, ids,
+                                                     context=context)],
+            context=context)
+        return self.write(cr, uid, ids, {'purged': True})
```
Comment (ajite) [inline database_cleanup/model/purge_menus.py]: You forgot to pass the context in this write method.

**Comment by guewen** [comment]
Once @ajite remark is  fixed, :+1:

File: database_cleanup/model/purge_menus.py (hunk 1 above)
Comment (hbrunn) [inline database_cleanup/model/purge_menus.py]: @ajite thanks, fixed

**Comment by lmignon** [comment]
:+1:

**Comment by sysadminmatmoz** [comment]
Just a remark.
When a PR changes or adds views, like:
-        'view/purge_menus.xml',

it should also change the module version (like 'version': '0.1', -> '0.1.1'), since an odoo restart is not enough in this case and the module has to be upgraded.

**Comment by pedrobaeza** [comment]
You're right. I forget to check that in my review...

**Pull request merged**